### ISSUE TITLE
Torrent download fails

### ISSUE THREAD

**Issue by rickfo97** [UNIT3D/UNIT3D#11]
- Laravel Version: 5.4.*
- PHP Version: 7.0.22-0
- Database Driver & Version: mysql 5.7.20-0

### Description:
If "public/files/tmp/" doesn't exist the download fails with file not found.

### Steps To Reproduce:
Have fresh install.
Upload torrent.
Try to download torrent from "[url]/download_check/slug.id"

### Possible Fix:
Have "public/flies/tmp" from install or add:
```PHP
if(!file_exists(getcwd() . '/files/tmp'))
            mkdir(getcwd() . '/files/tmp');
```
before the download.

**Comment by HDVinnie** [comment]
/public/files

is indeed missing the img , temp and torrents directories

**Comment by cbj4074** [comment]
Resolved via 21870c9e30735b4a8c318b32b3588d7966b90ef6 and f52af5fffec496c1a03e6d9e01a2aa0d4f05f968 .
